config/plugins/webhooks/demo/search/static/script.js

```diff
@@ -0,0 +1,1061 @@
+                self.registerToolLinkClick( self );
+            }
+        },
+
+        /** Build the fetched items template using the template dictionary */ 
+        makeToolSearchResultTemplate: function( collection, tool_template ) {
+	    var header_template = "",
+	        self = this,
+	        $el_search_result = $( '.search-results' ),
+	        $el_pin_item = null,
+	        $el_remove_item = null,
+                title = "Tools",
+                class_name = "search-section search-tools",
+                html = "",
+                header_text = "";
+            // Append header section when shown for all
+	    if( self.getActiveFilter() === "all" ) {
+	        header_text = self._buildHeaderTemplate( "tools", title, class_name );
+	    }
+            else {
+                header_text = self._buildHeaderTemplate( "tools", "", class_name );
+            }
+            // Make complete template
+	    _.each( collection, function( item ) {
+                html = html + item.template;
+	    });
+            html = html + tool_template;
+            self.setHTMLtoDOM( self, $el_search_result, '.search-tools', '.tool-search-link', html, header_text );
+        },
+
+        /** Set HTML text to DOM */
+        setHTMLtoDOM: function( self, $el, section_class, link_class , html_text, header_text ) {
+            if( html_text.length > 0 ) {
+                $el.append( header_text );
+                // Append the template to DOM
+                $el.find( section_class ).append( "<div>" + html_text +  "</div>" );
+                // Register link clicks for the new links
+	        self.registerToolLinkClick( self );
+	        self.registerLinkActionClickEvent( self, $( link_class ), $( '.search-tools' ) );
+            }
+            else {
+                if( self.getActiveFilter() !== "all" ) {
+                    $el.append( self._templateNoResults() );
+                }
+            }
+        },
+
+        /** Open the respective link as the modal pop up or in the center of the main screen */
+        searchedToolLink: function( _self, e ) {
+	    var id = "",
+	        form_style = "",
+	        version = "", 
+	        $target_element = null;
+	    if( e ) {
+	        _self.removeOverlay();
+	        // Set the target element as jQuery element
+	        if( e.srcElement ) {
+		    $target_element = $( e.srcElement );
+	        }
+	        else if( e.target ) {
+		    $target_element = $( e.target );
+	        }
+	        // Fetch the properties
+	        id = $target_element.attr( 'data-id' );
+	        form_style = $target_element.attr( 'data-formstyle' );
+	        version = $target_element.attr( 'data-version' );
+	        // Load as modal popup
+	        if( id === 'upload1' ) {
+		    Galaxy.upload.show();
+	        }
+	        // Open the link in the iframe
+	        else if ( form_style === 'regular' ) {
+		    var form = new ToolForm.View( { id : id, version : version } );
```

Comment:
If we set this link to document.location, it will reload the full page and after every reload, user needs to click on the search icon in masthead to activate the overlay search.